/**:
  ros__parameters:
    K: [ -162.836107, -41.319113, -120.870623, -16.811039]

    kd: 0.1
    ke: 0.1
    kp: 0.020

    # 0.0 angle - upward position. When absolute angle is less than this threshold
    # transition from swing up to LQR
    lqr_transition_angle: 0.523599
    rviz_test: false

    PID_KP: 2.0
    PID_KD: 2.0

    negative_clamp_limit: -100.0
    positive_clamp_limit: 100.0
   
# K_ = Eigen::Vector4d(this->get_parameter("K").as_double_array().data());
# ke_ = this->get_parameter("ke").as_double();
# kv_ = this->get_parameter("kv").as_double();
# kx_ = this->get_parameter("kx").as_double();
# kdelta_ = this->get_parameter("kdelta").as_double();
# lqr_transition_angle_ = this->get_parameter("lqr_transition_angle").as_double();
#
#    double PID_k_;
#   double PID_d_;